client.go

```diff
@@ -11,25 +10,32 @@ import (
 
 // Setup the HTTP client and response functionality
 const APIBaseUrl = "https://app.tuneuptechnology.com/api/"
-// const UserAgent = "Tuneup_Technology_App/GoClient/" + Version // TODO: Implement this
+// const UserAgent = "TuneupTechnologyApp/GoClient/" + Version // TODO: Implement this
```

Comment:
Complete this TODO?

Reply:
Moved to an issue instead of commented code.